Unable to configure format using the maven plugin

The maven plugin is unable to convert a configured format to an enum constant of Format. 
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.649 s
[INFO] Finished at: 2019-04-16T18:40:52+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.korthout:cantis:0-SNAPSHOT:generate (cantis) on project cantis-usage-example: Unable to parse configuration of mojo com.github.korthout:cantis:0-SNAPSHOT:generate: Cannot convert 'json' to Enum: No enum constant com.github.korthout.cantis.formatting.Format.json -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginConfigurationException
```

--- a/src/main/java/com/github/korthout/cantis/maven/Generate.java
+++ b/src/main/java/com/github/korthout/cantis/maven/Generate.java
@@ -58,7 +58,7 @@ public final class Generate extends AbstractMojo {
      * Output format. Defaults to {@code Format.PLAIN}.
      */
     @Parameter(name = "format", defaultValue = "plain")
-    private Format format;
+    private String format;
 
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
@@ -71,7 +71,7 @@ public final class Generate extends AbstractMojo {
                 this.target.isBlank()
                     ? new ToLog(log)
                     : new ToFile(this.target),
-                this.format
+                Format.fromString(this.format)
             ).print();
         } catch (final IllegalArgumentException exception) {
             log.warn(String.format("Directory %s not found.", this.source));

